Testers/Annotations/double_mustcall/index.txt

```diff
@@ -0,0 +1,3 @@
+MustCall target = STRUCT(M).FIELD(0) methods = free, free1
```

Comment:
As a general rule, anything that can be automatically generated shouldn't be checked into source control. This is also why e.g., we don't check in Java `.class` files or `.o` object files. 

These `.txt` files look automatically generated to me (in particular, the new ones in this PR, like this file, that reference fields by index). Can we avoid checking them in and instead automate their production?

Reply:
The files are not automatically generated and serve as the ground truth/reference for what the annotation generator should output. The annotation tester works by running the current annotation generator and running `diff` on its current output and the corresponding file in `Testers/Annotations`